template/php/Dockerfile

```diff
@@ -0,0 +1,52 @@
+# start with the official Composer image and name it
+FROM composer:1.7.1 AS composer
+
+# continue with the official PHP image
+FROM php:7.2-alpine
+
+# copy the Composer PHAR from the Composer image into the PHP image
+COPY --from=composer /usr/bin/composer /usr/bin/composer
+
+# Composer requirements
+ARG COMPOSER_AUTH='{}'
+ENV COMPOSER_AUTH=${COMPOSER_AUTH}
+
+# Add git
+RUN apk add --no-cache git
```

Comment:
I'd like to see all of those instructions combined with `&&` for the benefit of caching.

Reply:
This would be best moved into the `RUN` step where the `app` user and homedir are created.